tests/testthat/test_04b_weather_raw.R

```diff
@@ -0,0 +1,20 @@
+context("weather_raw with vcr")
```

Comment:
I ran the test once to create the cassette, then I turned off wifi and re-ran the test to convince myself it hard worked.